client/v2/doc.go

```diff
@@ -0,0 +1,6 @@
+// Lumberjack protocol version 2 client implementations.
```

Comment:
Shouldn't this comment begin with `Package v2...`? (same for other package level comments)